peach/src/main/java/com/hartwig/hmftools/peach/event/HaplotypeEvent.java

```diff
@@ -0,0 +1,13 @@
+package com.hartwig.hmftools.peach.event;
+
+import org.jetbrains.annotations.NotNull;
+
+public interface HaplotypeEvent
```

Comment:
There's only one haplotype event right now (the `VariantHaplotypeEvent`). I guess there will be more types in the future?

Reply:
Yes, there will be also be events for deletions and likely fusions. But this is out of scope for the first Peach Java release.